README.md

```diff
@@ -137,9 +137,8 @@ The yarn build scripts default to production for libraries if `PANOPTES_ENV` is
 
 ### Docker images
 
-- `zooniverse/front-end-monorepo`: Built from the Dockerfile in the root directory. It runs `yarn install` and `yarn build` for the `lib-` packages.
-- `zooniverse/fe-content-pages`: Built from the Dockerfile in `packages/app-content-pages`. Runs `yarn build` and `yarn start` in `packages/app-content-pages` from `zooniverse/front-end-monorepo:latest`.
-- `zooniverse/fe-project`: Built from the Dockerfile in `packages/app-project`. Runs `yarn build` and `yarn start` in `packages/app-project` from `zooniverse/front-end-monorepo:latest`.
+- `zooniverse/front-end-monorepo-staging`: Built from the Dockerfile in the root directory. It runs `yarn install` and builds all the libraries and apps from the latest main branch commit.
+- `zooniverse/front-end-monorepo-production`: Built from the Dockerfile in the root directory. It runs `yarn install` and builds all the libraries and apps from the `production_release` tag.
```

Comment:
Do we need two images, now that the monorepo builds are tagged with the commit hash? Can we use `zooniverse/front-end-monorepo:${GIT_COMMIT}` instead when we deploy to staging and to production?

Reply:
I have a vague memory that we switched from deploying `fe-project:${GIT_COMMIT}` to deploying `fe-project-staging:${GIT_COMMIT}` and `fe-project-production:${GIT_COMMIT}` because of a problem where requests for JS files 404'ed in production, but I don't remember the details. Maybe having separate images for staging and production deploys is safer?

Reply:
I remember now: a staging build and a production build from the same commit ID will build different JS bundles. If we only use commit IDs to identify builds, we can run into a situation where we're accidentally trying to use a staging build with the production HTML, resulting in 404 errors for the JS files.

Reply:
Should befine. Do we still want a build that pulls the production image from Docker Hub and runs it locally? Local `docker-compose` will build and run from the latest local HEAD commit.

Reply:
we don't have these for any other apps and can add a custom docker-compose file / manually set it up if needed.

Reply:
👍 

For the benefit of anyone else following this conversation, with this PR `docker-compose up --build` should build and run a local image which matches the Jenkins build (except for running behind a proxy.)

If we were to add a proxy service, as part of the local compose setup, then that could catch bugs like #1826 when we run the apps locally in Docker.